rollbar-reactive-streams/src/test/java/com/rollbar/reactivestreams/UtilsTest.java

```diff
@@ -0,0 +1,112 @@
+package com.rollbar.reactivestreams;
+
+import java.util.stream.IntStream;
+import java.util.stream.Stream;
+import org.reactivestreams.Publisher;
+import org.reactivestreams.tck.PublisherVerification;
+import org.reactivestreams.tck.TestEnvironment;
+import org.testng.annotations.Ignore;
+import org.testng.annotations.Test;
+import reactor.core.publisher.Flux;
+
+public class UtilsTest {
```

Comment:
What is a purpose of `UtilsTest` if all tests cases are empty or ignored?

Reply:
38 tests are inherited from the base class, provided by the reactivestreams project. We only override the tests that we need to disable because they're not applicable.